generator-core/pom.xml

```diff
@@ -5,24 +5,21 @@
     <parent>
         <artifactId>spring-boot-cli</artifactId>
         <groupId>br.com.netodevel</groupId>
-        <version>0.0.1.BUILD-SNAPSHOT</version>
+        <version>0.0.2.BUILD-SNAPSHOT</version>
```

Comment:
Consider using a more specific versioning scheme, such as Semantic Versioning, instead of `0.0.2.BUILD-SNAPSHOT` to ensure better version control and tracking.